package.json

```diff
@@ -17,7 +17,7 @@
 		"prettier": "turbo run prettier",
 		"lint": "turbo run lint",
 		"pretest": "npm run clean && npm run build",
-		"test": "vitest run",
+		"test": "NODE_OPTIONS='--es-module-specifier-resolution=node' vitest run",
```

Comment:
I don't love this, but ckohen tells me to not need it I gotta make my code ugly sadface emoji

Reply:
That implies your code isn't ugly already 😆